images/postgresql/02_data.sql

```diff
@@ -0,0 +1,6 @@
+-- Values from daemon's Dockerfile
+-- Needs `id >= 32` for non-rfc php versions
+INSERT INTO public."version" ("id", "name", "command")
+	VALUES (32, 'alpine-latest','/usr/bin/php -c /etc -q');
+
+-- TODO: Insert users ?
```

Comment:
I do not know if that would be relevant. I do not really know what can those `users` do.

Reply:
the `users` are an unused feature so you can ignore it, that's fine